src/engraving/dom/slur.cpp

```diff
@@ -120,9 +122,15 @@ bool SlurSegment::edit(EditData& ed)
 
     if (ed.key == Key_Home && !ed.modifiers) {
         if (ed.hasCurrentGrip()) {
-            ups(ed.curGrip).off = PointF();
+            startEditDrag(ed);
+            if (ed.curGrip == Grip::SHOULDER) {
```

Comment:
"shoulder" is a special case grip where any offset applied to it is actually transferred to the other curve points, so resetting those should give essentially the expected behaviour in most cases